### PR TITLE
fix: [MM-2057] Fixed usage of hide option for NavbarAccordion

### PR DESCRIPTION
**Pull request by SiriusCrain**

### CODE REVIEW

File: packages/lab/src/navbar/NavbarAccordion.tsx [hunk 1]
```diff
@@ -118,6 +119,18 @@ export function NavbarAccordion({
...
 
+  const filteredItems: NavbarItemOptions[] = useMemo(
+    () =>
+      items
+        .filter((item) => {
+          return !item.hide;
+        })
+        .map((item) => ({
+          ...item,
+        })),
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarAccordion.tsx:130]:
```suggestion
        .filter((item) => !item.hide),
```

File: packages/lab/src/navbar/NavbarAccordion.tsx [hunk 2]
```diff
@@ -118,6 +119,18 @@ export function NavbarAccordion({
     setExpanded(isNavbarExpanded);
   }, [isNavbarExpanded]);
 
+  const filteredItems: NavbarItemOptions[] = useMemo(
```
Comment (doniyor2109) [inline packages/lab/src/navbar/NavbarAccordion.tsx:122]:
```suggestion
  const filteredItems:Array<Omit<NavbarItemOptions, 'icon'>> = useMemo(
```

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: packages/lab/src/navbar/NavbarAccordion.tsx (hunk 1 above)
Comment (SiriusCrain) [reply packages/lab/src/navbar/NavbarAccordion.tsx]: Fixed

File: packages/lab/src/navbar/NavbarAccordion.tsx (hunk 2 above)
Comment (SiriusCrain) [reply packages/lab/src/navbar/NavbarAccordion.tsx]: Fixed

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**